src/css/parse.js

```diff
@@ -3257,8 +3257,12 @@ function getNamespacePrefix() {
 function checkNamespaceSeparator(i) {
   if (i >= tokensLength) return 0;
 
-  if (tokens[i].type === TokenType.VerticalLine) return 1;
-  else return 0;
+  if (tokens[i].type === TokenType.VerticalLine) {
+    // Return false if `|=` - [attr|=value]
+    return (tokens[i + 1] && tokens[i + 1].type === TokenType.EqualsSign) ? 0 : 1;
+  }
```

Comment:
Sorry about that. I've implemented the standard way for all 3 (CSS, Sass, SCSS). I also modified the conditional slightly to make it easier to read and improve code flow.